packages/fuels/src/cli/commands/build/forcHandlers.ts

```diff
@@ -6,8 +6,7 @@ type OnErrorFn = (reason?: number | Error) => void;
 export const onForcExit =
   (onResultFn: OnResultFn, onErrorFn: OnErrorFn) => (code: number | null) => {
     if (code) {
-      onErrorFn(code);
-      //  process.exit()?
+      onErrorFn(new Error(`forc exited with error code ${code}`));
```

Comment:
The `onErrorFn` would be a `reject`, and previously it'd reject with `1`, which isn't possible to re-throw in the `withConfigErrorHandler` as I [changed](https://github.com/FuelLabs/fuels-ts/pull/3504/files#diff-004a628a5877d71458702addd56c030e11787ec9dccea78251d8952d2c39e36bR11) to be done now.